src/repairnator-core/src/main/java/fr/inria/spirals/repairnator/dockerpool/BuildAnalyzerLauncher.java

```diff
@@ -6,7 +6,7 @@
 import com.martiansoftware.jsap.JSAPResult;
 import fr.inria.spirals.repairnator.InputBuildId;
 import fr.inria.spirals.repairnator.LauncherType;
-import fr.inria.spirals.repairnator.LauncherUtils;
```

Comment:
We avoid moving classes, it easily breaks client code. Could you move it back? Thanks!

Reply:
Done!